### ISSUE TITLE
Error: Unknown encoding binary

### ISSUE THREAD

**Issue by johnkawakami** [chadxz/imap-simple#14]
I get this message when I'm decoding the DMARC reports message from Yahoo. I get it when I call getPartData() and try to get this part of the message:

```
--_----------=_146780861314197191
Content-Disposition: inline
Content-Length: 44
Content-Transfer-Encoding: binary
Content-Type: text/plain

This is an aggregate report from Yahoo! Inc.
```

[According to RFC 1341, encoding binary is illegal.](https://www.w3.org/Protocols/rfc1341/5_Content-Transfer-Encoding.html)
Despite this, they do specify that "binary" is an encoding.

I'm going to try and fix the library.

**Comment by chadxz** [comment]
> I'm going to try and fix the library

Sounds good, let me know if I can help in any way

**Comment by chadxz** [comment]
closed by #15
